terraform/environment/modules/environment/ecs_feedbackdb.tf

```diff
@@ -0,0 +1,97 @@
+//----------------------------------
+// The Api service's Security Groups
+#tfsec:ignore:AWS018 - Adding description is destructive change needing downtime. to be revisited
+resource "aws_security_group" "feedbackdb_ecs_service" {
+  name_prefix = "${terraform.workspace}-feedbackdb-ecs-service"
+  vpc_id      = data.aws_vpc.default.id
+  tags        = merge(local.default_opg_tags, local.feedbackdb_component_tag)
+}
+
+//----------------------------------
+// Anything out except production
+#tfsec:ignore:AWS018 - Adding description is destructive change needing downtime. to be revisited
+resource "aws_security_group_rule" "feedbackdb_ecs_service_egress" {
+  count     = var.environment_name == "production" ? 0 : 1
+  type      = "egress"
+  from_port = 0
+  to_port   = 0
+  protocol  = "-1"
+  #tfsec:ignore:AWS007 - anything out
+  cidr_blocks       = ["0.0.0.0/0"]
+  security_group_id = aws_security_group.feedbackdb_ecs_service.id
+}
+
+//--------------------------------------
+// feedbackdb ECS Service Task level config
+
+resource "aws_ecs_task_definition" "feedbackdb" {
+  count                    = var.environment_name == "production" ? 0 : 1
```

Comment:
ah spotted another issue - this doesn't run in production

Reply:
fixed